src/app/pages/partners/partner-details/reviews/user-review.js

```diff
@@ -13,19 +13,23 @@ function UserControls({status}) {
     const {postRating} = usePartnerContext();
     const myReview = useMyReview();
     const displayStatus = ['Deleted', 'Rejected'].includes(status) ? status : 'pending';
-
-    function onUpdate(event) {
-        event.preventDefault();
-        togglePage();
-    }
-
-    function onDelete(event) {
-        event.preventDefault();
-        // eslint-disable-next-line no-alert
-        if (window.confirm('Are you sure you want to delete this review?')) {
-            postRating({id: myReview.id}, 'DELETE');
-        }
-    }
+    const onUpdate = React.useCallback(
```

Comment:
FYI, tutor had a nifty `SuretyGuard` component that overlays a button with an "are you sure" popup: https://github.com/openstax/tutor-js/blob/main/shared/src/components/surety-guard.js

I'll probably steal it in kinetic when/if we have a need for it